packages/neuron-wallet/src/controllers/wallets.ts

```diff
@@ -178,6 +217,13 @@ class WalletsController {
         }
         if (newPassword) {
           if (WalletsController.service.validate({ id, password })) {
+            if (!WalletsController.verifyPasswordComplexity(newPassword)) {
+              return {
+                status: ResponseCode.Fail,
+                msg:
+                  'At least 8 characters, three types of uppercase and lowercase alphabets, numbers, and special symbols!',
```

Comment:
The message appears in multiple places, and is not localized.

Reply:
ok